src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -241,7 +242,10 @@ public PlatformDetectorResult Detect(RepositoryContext context)
                 compressVirtualEnvCommand: compressVirtualEnvCommand,
                 compressedVirtualEnvFileName: compressedVirtualEnvFileName,
                 runPythonPackageCommand: isPythonPackageCommandEnabled,
+                pythonManifestFileName: Path.Join(manifestDirPath, "oryx-build-commands.txt"),
```

Comment:
same for other places like this

Reply:
Consider generalizing pythonManifestFileName and nodeManifestFileName to a single one, for other platforms such as ruby, php.

Reply:
yes I will change it to a generalized variable name and a constant for buildcommand file name